### ISSUE TITLE
[Quest] "Princess Must Die!" QuestID-88 Incorrect

### ISSUE THREAD

**Issue by Dexetren** [dalaranwow/dalaran-wow#5749]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Princess and her bodyguards are currently stationary with no waypoint to follow.  The mobs themselves are also not leashed and can be engaged individually without pulling one another.  Also, there is one too many bodyguards as one was removed as stated per various comments...

_"By pogostick (1,296 – 5·20) on 2007/11/17 (Patch 2.3.0) - As of patch 2.3 they removed 1 bodyguard. So, it's now just Princess (lvl 9) and a single bodyguard (lvl 7)."_

_"Z4kW0lf on 2009/10/19 (Patch 3.2.2) - As of patch 2.3,now she have only 1 bodyguard."_

**Expected behaviour**: Princess should be roaming the pumpkin patch in a circle with her bodyguards following.  There should only be one bodyguard.  And upon engaging combat with either Princess or the bodyguard both mobs should aggro as they are leashed.

**Steps to reproduce the problem**:

1. Obtain "Princess Must Die!"(88) from Ma Stonefield (244)
2. Go to pumpkin patch
3. Engage Princess and her bodyguards.

**Include proofs for this behaviour**
![Screenshot (752)](https://user-images.githubusercontent.com/52165800/60024328-94927400-9665-11e9-88d9-d21a1bed90c4.png)
![Screenshot (753)](https://user-images.githubusercontent.com/52165800/60024334-98be9180-9665-11e9-8982-ec98fac9bdd6.png)

per wowhead comments https://www.wowhead.com/quest=88/princess-must-die#comments

_"By pogostick (1,296 – 5·20) on 2007/11/17 (Patch 2.3.0) - As of patch 2.3 they removed 1 bodyguard. So, it's now just Princess (lvl 9) and a single bodyguard (lvl 7)."_

_"Z4kW0lf on 2009/10/19 (Patch 3.2.2) - As of patch 2.3,now she have only 1 bodyguard."_

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
